src/tools/generate/index.js

```diff
@@ -0,0 +1,5 @@
+let { generate_chapters } = require('./generate_chapters');
+
+// TODO: Generate visualisations 
+generate_chapters();
```

Comment:
Could you also add docs to the README with steps to install and use the generation script? You can add it under a new section "Generating chapters" since it's distinct from running the website locally.